molgenis-apps/src/main/resources/templates/view-apps.ftl

```diff
@@ -1,50 +0,0 @@
-<#include "molgenis-header.ftl">
-<#include "molgenis-footer.ftl">
-
-<#assign css=["apps.css"]>
-<#assign js=["apps.js"]>
-
-<@header css js/>
-<@hasPermission entityTypeId='${appEntityTypeId}' permission="WRITE">
-<div class="row header">
-    <div class="col-md-12">
-        <button class="btn btn-default" id="create-app-btn">Create new App</button>
-    </div>
-</div>
-</@hasPermission>
-<div class="row">
-<#list apps as app>
-    <#if app?index gt 0 && app?index % 4 == 0>
```

Comment:
why 4 ? is it related to screen size ? , maybe used named const or more dynamic method